mkdocs.yml

```diff
@@ -2,7 +2,7 @@
 # See: https://www.mkdocs.org/user-guide/configuration/
 site_name: Coda Packs SDK
 # TODO(spencer): Update with new url
-site_url: !ENV [MK_DOCS_SITE_URL, "https://coda.io/developers/temp-developer-docs/packs/latest/"]
```

Comment:
Was this intentional?

Reply:
yeah meant to comment on this but this makes the local mkdocs serving work by defaulting to null when this isn't set